js/dogs-jquery.js

```diff
@@ -0,0 +1,13 @@
+$('.grey-border').on('mouseover', function (e) {
```

Comment:
One of the cool things about this, is you may not need to remove the grey-border class as the class you add in on hover might override it. But a good thought process.